include/ibamr/FEMechanicsBase.h

```diff
@@ -394,6 +394,15 @@ class FEMechanicsBase : public SAMRAI::tbox::Serializable
                                              VolumetricEnergyDerivativeFcn d2U_dJ2_fcn = nullptr,
                                              unsigned int part = 0);
 
+    /*!
+     * Indicate whether there is a static or dynamic pressure associated with the specified FE mesh part.
+     */
+    inline bool partHasPressure(unsigned int part = 0)
```

Comment:
Methods defined in the class declaration are always inline (we don't have to sweat the language-lawyer stuff - this is a PSA)